types/react-table/index.d.ts

```diff
@@ -1,12 +1,12 @@
-// Type definitions for react-table 6.7
+// Type definitions for react-table 6.8
```

Comment:
Don't bump this unless the library version changed.

Reply:
Yep, I just want to trigger build on CI.